kotlin/build.gradle

```diff
@@ -206,3 +210,31 @@ subprojects {
     ]
   }
 }
+
+// We use JetBrain's Kotlin Binary Compatibility Validator to track changes to our public binary
+// APIs.
+// When making a change that results in a public ABI change, the apiCheck task will fail. When this
+// happens, run ./gradlew apiDump to generate updated *.api files, and add those to your commit.
+// See https://github.com/Kotlin/binary-compatibility-validator
+apply plugin: 'binary-compatibility-validator'
```

Comment:
I think you could consider moving this to a separate gradle file if you think this build.gradle will keep growing, but doesn't seem like the pattern right now anyways (based on this file).

Reply:
Great idea! I'll do that now to keep this file from getting bloated.